packages/graphback-datasync/src/DataSyncPlugin.ts

```diff
@@ -133,4 +136,25 @@ export class DataSyncPlugin extends GraphbackPlugin {
       return dataSyncService.sync(args.lastSync);
     }
   }
+
+  protected addMetaFields(schemaComposer: SchemaComposer<any>, models: ModelDefinition[]) {
```

Comment:
The function name could be `addDataSyncMetaDataFields` considering my suggestions above